services-geocoding/src/main/java/com/mapbox/api/geocoding/v5/MapboxGeocoding.java

```diff
@@ -607,6 +628,22 @@ public MapboxGeocoding build() {
         && geocoding.limit() != null && !geocoding.limit().equals("1")) {
         throw new ServicesException("Limit must be combined with a single type parameter");
       }
+
+      if (intersectionStreets.size() == 2) {
+        if (!(geocoding.mode().equals(GeocodingCriteria.MODE_PLACES)
+                || geocoding.mode().equals(GeocodingCriteria.MODE_PLACES_PERMANENT))) {
+          throw new ServicesException("Geocoding mode must be " + GeocodingCriteria.MODE_PLACES
+                  + " or " + GeocodingCriteria.MODE_PLACES_PERMANENT + " for intersection search.");
```

Comment:
"Geocoding mode must be GeocodingCriteria.MODE_PLACES or GeocodingCriteria.MODE_PLACES_PERMANENT for intersection search." 

Include `GeocodingCriteria.MODE_PLACES` & `GeocodingCriteria.MODE_PLACES_PERMANENT` in the actual exception message?


Same thing as my comment about `GeocodingCriteria.TYPE_ADDRESS`.

Reply:
Sure.